zerver/views/registration.py

```diff
@@ -137,6 +137,8 @@
     password_auth_enabled,
 )
 
+logger = logging.getLogger("")
```

Comment:
This should have a commit message explaining why?

Reply:
yeah, should be helpful to explain:
```
Having a logger object will be helpful in the upcoming commit to pass as
argument to sync_groups. We use the root logger here (sticking to
original behavior) rather than setting up `getLogger(__name__)` to avoid
breaking a large number of tests which assert logs content - as there
doesn't seem to be any major upside to using a separate logger for this file.
```